en/middleware/xrce_dds.md

```diff
@@ -180,17 +180,25 @@ For more information about setting up communications channels see [Pixhawk + Com
 The XRCE-DDS client module ([microdds-client](../modules/modules_system.md#microdds-client)) is included by default in all firmware and the simulator.
 This must be started with appropriate settings for the communication channel that you wish to use to communicate with the agent.
 
```

Comment:
```suggestion
@beniaminopozzan I think it is worth saying this up-front still. Otherwise you might think you need to do something on the simulator.

:::note
The simulator automatically starts the client on localhost UDP port `8888` using the default microdds namespace.
:::

```